.changeset/chilled-comics-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+"@tevm/contract": patch
+---
+
+Updated types to NodeNext for maximum compatibility with all build tool setups
```

Comment:
```suggestion
---
"@tevm/contract": patch
---

Updated types to NodeNext for maximum compatibility with all build tool setups. NodeNext requires explicit file extensions like `./Foo/index.js` instead of `./Foo` which maximizes general compatibility with latest versions of node and all bundlers.
```